README.md

```diff
@@ -118,8 +118,33 @@ Both CI systems build all merges to master as well as pull requests.
 The repository contains a script [`setupLocalProxy.sh`](./scripts/setupLocalProxy.sh) which can be used to run the Proxy locally.
 A prerequisite for this is access to https://console.rh-idev.openshift.com/.
 To run the script you need to export your OpenShift access token for console.rh-idev.openshift.com as DSAAS_PREVIEW_TOKEN.
-Note, you need edit permissions on the dsaas-preview namespace in order to port forward.
-
+Note, you need edit permissions on the dsaas-preview namespace in order to port forward. 
```

Comment:
Note, In order to port forward you need to edit permissions on the dsaas-preview namespace